src/backend/gl/src/native.rs

```diff
@@ -15,13 +17,16 @@ pub type FrameBuffer = gl::types::GLuint;
 pub type Surface     = gl::types::GLuint;
 pub type Texture     = gl::types::GLuint;
 pub type Sampler     = gl::types::GLuint;
+pub(crate) type SetID     = gl::types::GLuint;
```

Comment:
the whole module is private to the crate, so no need to have `pub(crate)` for all the things here, just `pub` should be good